.github/workflows/auto-approve.yml

```diff
@@ -5,6 +5,8 @@ jobs:
   build:
     runs-on: ubuntu-latest
     name: Test changed-files
+    outputs:
+      output1: ${{ steps.changed-files-specific.outputs.any_changed }}
```

Comment:
Let's change the name for something more descriptive. Instead of `output1`, maybe `has_go_mod_changes` or something like this?

Reply:
Sure, wanted to test first but let me change it :)